mayhem/Mayhemfile

```diff
@@ -0,0 +1,12 @@
+project: stegano
```

Comment:
Please omit image name here. Any reason for the 8 second timeout?

Reply:
The timeout is not needed, I will remove it. Should I remove image name on all of my PRs?